README.md

```diff
@@ -99,138 +99,157 @@ outputs, and adhere to any given constraints.
  - Add install directory `C:\Program Files (x86)\Graphviz2.38\bin` to PATH
  
 3. Run the following as admin to configure GraphViz:
-```
-dot -c
-```
+   ```
+   dot -c
+   ```
 
 4. Clone repo (in directory of you choosing)
-```console
-git clone https://github.com/ScottJDaley/ada.git
-```
+   ```cmd
+   git clone https://github.com/ScottJDaley/ada.git
+   ```
 
 5. Move inside the repo
-```console
-cd ada
-```
+   ```cmd
+   cd ada
+   ```
+
+5. Create virtual environment and activate it
+
+   ```cmd
+   py -m venv venv && venv\Scripts\activate
+   ```
+
+   :bulb: You may deactivate the virtual environment with `deactivate`
 
 6. Install python modules
-```console
-py -m pip install -r requirements.txt
-```
+   ```cmd
+   py -m pip install -r requirements.txt
+   ```
 
 7. Start the bot
-```
-py bot.py
-```
+   ```cmd
+   py bot.py
+   ```
 
 ### MacOS Installation
 
-#### Requirements
-- Homebrew
-- python3 3.9+
-- python3's pip
-- Graphviz
+1. Download and install [Homebrew](https://brew.sh/)
 
-1. Create python virtual environment
-```bash
-python3 -m venv venv
-source ./venv/bin/activate
-```
+   :bulb: Make sure Hombrew's package lists are up-to-date with `brew update`
 
-2. Install python requirements to venv
-```bash
-pip3 install -r requirements.txt
-```
+2. Install Python 3
+   ```zsh
+   brew install python3
+   ```
 
 3. Install GraphViz
-```bash
-brew install graphviz
-```
-
-4. Install python packages
-```bash
-pip install -r requirements.txt
-```
-
-5. Set up Discord bot
-```bash
-export DISCORD_TOKEN={token from discord developer portal}
-```
-
-6. Start the bot
-```bash
-python bot.py
-```
+   ```zsh
+   brew install graphviz
+   ```
+
+1. Clone repo (in directory of you choosing)
```

Comment:
The numbering here keeps restarting. Pretty sure you can just do 1. for all the items in a markdown list.